### DISCUSSION TITLE
I am blocked from logining in

### DISCUSSION THREAD

**Discussion by Deocoder** [vercel/community#4083] [Account Issue] [answered]
### Summary

Says this account is blocked , when trying to log in in with my github account.
Pls help

### Steps to Reproduce

Pls help me i want deploy a webpage

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble logging in. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Reply by oluiscabral** [reply]
I'm facing the same issue. I think I've never created a Vercel account before. What can I do to solve this? And I didn't receive any e-mail...

**Reply by starry-wang** [reply]
me too, can u help me ?
